src/content/en/fundamentals/primers/service-workers/what-do-they-do.markdown

```diff
@@ -0,0 +1,38 @@
+---
+layout: shared/narrow
+published: false
+title: "What do they do?"
+description: "Site search is vital for helping mobile users find what they’re looking for in a hurry"
+authors:
+- josephmedley
+published_on: 2015-10-01
+updated_on: 2015-10-01
+order: 2
+---
+
+<p class="intro">
+  Service workers provide web pages with capabilities that make them more like 
+  native apps. Here are a list of characteristics and apps or sites that 
+  demonstrate their use.
+</p>
+
+### Responsiveness
```

Comment:
I think if you're going to make an argument about responsiveness benefits, you need to bring up the concept of an app shell, and how service workers facilitate serving that shell almost immediately.